doc/src/explore/devnet.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Experiment with Sui DevNet
+---
+
+Welcome to the beginnings of the Sui DevNet. It exists now to gain operational experience with the Sui software in a public setting. The Sui DevNet currently consists of:
+
+* A four-validator network with all nodes operated by Mysten Labs. Clients send transactions and read requests via this endpoint: http://gateway.devnet.sui.io/
+* A public network [Sui Explorer](https://github.com/MystenLabs/sui/tree/main/explorer/client#readme) for browsing the TestNet transaction history: http://explorer.devnet.sui.io
+* A [Discord channel](https://discordapp.com/channels/916379725201563759/971488439931392130) for requesting test coins that can be used to pay for gas on the test network. These coins have no financial value and will disappear each time we reset the network.
```

Comment:
meaning we just add a wallet cli bullet point here and do not mention "faucet" since it's not accessible to external users.

Reply:
We have other references to faucet here. How should we handle them?

Reply:
Where are the other references? I tried to search "faucet" but did not find any.

Reply:
> Where are the other references? I tried to search "faucet" but did not find any.

Disregard. That was in the draft.